src/libraries/Microsoft.PowerFx.Interpreter/Functions/Mutation/RemoveFunction.cs

```diff
@@ -209,9 +206,11 @@ public async Task<FormulaValue> InvokeAsync(FormulaValue[] args, CancellationTok
             var recordsToRemove = args.Skip(1).Take(args.Length - toExclude);
 
             cancellationToken.ThrowIfCancellationRequested();
-            var ret = await datasource.RemoveAsync(recordsToRemove, all, cancellationToken).ConfigureAwait(false);
-
-            return ret.ToFormulaValue();
+            var ret = await datasource.RemoveAsync(recordsToRemove, all, cancellationToken).ConfigureAwait(false);
+
+            // If the result is an error, propagate it up. else return blank.
+            var result = ret.IsError ? ret.ToFormulaValue() : FormulaValue.NewBlank();
```

Comment:
>var result = ret.IsError ? ret.ToFormulaValue() : FormulaValue.NewBlank(); [](http://example.com/codeflow?start=12&length=74)

Doing this to avoid breaking public surface on RemoveAsync